### PR TITLE
Add project name(s) and ID(s) to tasks

### PR DESCRIPTION
**Pull request by irvingpop**
**Are you a current Fivetran customer?** 

Yes
Irving Popovetsky, Director of Customer Engineering at Honeycomb.io

**What change(s) does this PR introduce?** 

This PR won't modify any existing outputs, but will add 5 new columns to the `asana__task` model:
1. project_id (the "first" seen/synced project associated with a task)
2. project_ids (a comma-separate list of project IDs)
3. project_name
4. project_names
5. number_of_projects

**Did you update the CHANGELOG?** 

- [ ] Yes
(not yet, but I'll push a CHANGELOG entry after initial review)

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes
(not yet, but I'll push a version bump entry after initial review)

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [link bug/feature number here](https://github.com/fivetran/dbt_asana/issues/24)
- [ ] No 
Fixes: #24 

**How did you test the PR changes?** 

 

- [ ] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [x] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dizzy:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/asana__task.sql [hunk 1]
```diff
@@ -5,7 +5,7 @@ with task as (
 
 task_comments as (
 
-    select *
```
Comment (irvingpop) [inline models/asana__task.sql:8]: Apologies for the extra diffs, it was auto whitespace trimming in my editor.  I can put these back if you want!

File: dbt_project.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ models:
       +materialized: ephemeral
 vars:
   asana:
-    project_task: "{{ ref('stg_asana__project_task') }}"
+    project_task: "{{ ref('stg_asana__project_task_tmp') }}"
```
Comment (irvingpop) [inline dbt_project.yml:13]: I'm not sure if this is the best way to handle this, but I wanted access to the `_fivetran_synced` column to help set the "primary" project in a repeatable way.  I'm open to other suggestions!

**Comment by fivetran-catfritz** [comment]
Hi @irvingpop, thank you for your PR contribution, and thank you for your patience in waiting for our response! I took a quick look through your changes and think these would great columns to bring into the package. We have scheduled this for a more thorough review later this month. 

In the meantime, one thing that stands out to adjust is how to bring in `_fivetran_synced`. We prefer not to change how our variables are defined in `dbt_project.yml`, so it would be best to add `_fivetran_synced` to the [source table itself](https://github.com/fivetran/dbt_asana_source/blob/c19b7248610652ad724c350006db0b47b5537899/models/stg_asana__project_task.sql#L22). This way it will persist `stg_asana__project_task` in the transform without hassle. We can make these changes when we do our final review, but if you are able to update this PR and create a second PR on the source with this change, it will help streamline the final review!

**Comment by fivetran-catfritz** [comment]
Hi @irvingpop To keep you in the loop I have a couple updates to make, so I will merge your branch into a newly created one, and then go from there. I'll let you know when it's ready for testing!

**Pull request merged**